packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint import/no-unresolved: off */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import './LineGraph.module.scss';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.array,
+  /**
+   * timeout to display multiple data contents in specific time interval.
+   */
+  timeout: PropTypes.array,
+};
+
+const LineGraph = ({
+  graphConfig, dataset, graphID, timeout,
+}) => {
+  React.useEffect(() => {
+    const graph = Carbon.api.graph(graphConfig);
+    if (!(utils.isUndefined(dataset))) {
+      if (!(utils.isUndefined(timeout))) {
+        let count = 0;
+        dataset.forEach((data) => {
+          setTimeout(
+            () => (graph.graphContainer
+              ? graph.loadContent(Carbon.api.line(data))
+              : ''),
+            timeout[count],
+          );
+          count += 1;
+        });
+      } else {
+        dataset.forEach((data) => {
+          graph.loadContent(Carbon.api.line(data));
+        });
+      }
+    }
+  }, [graphConfig, dataset, timeout]);
+
+  return (
+    <div id="canvasContainer">
+      {/* eslint-disable-next-line react/forbid-dom-props */}
+      <div id="tooltip" className="tooltip" style={{ display: 'none' }} />
```

Comment:
We should avoid using inline styles if possible.

Reply:
Removed inline styles [here](https://github.com/cerner/terra-graphs/pull/106/commits/1eef90b7c4038e57ab95b103e1db004b45318196#diff-21099d22317b65a81c95ece3f22955d83ab8a2ede2ccbde8f2c32ab64cb8b1eaR37).